mainnet/03cbbe1cfdd4f92712e9921c2bfecdf5ab1041436d799e5d96c4a407326cb2cf0f.json

```
@@ -0,0 +1,9 @@
+{ “id”: 59,
```

This is invalid JSON.

```suggestion
{
  "id": 59,
```